pkg/config/env/environment_containers.go

```diff
@@ -248,6 +250,23 @@ func detectPodResources(features FeatureMap, cfg model.Reader) {
 	}
 }
 
+func detectDevicePlugins(features FeatureMap, cfg model.Reader) {
+	// We only check the path from config if the path from config exists (does not have unix:// prefix)
+	socketDir := cfg.GetString("kubernetes_kubelet_deviceplugins_socketdir")
+	stat, err := os.Stat(socketDir)
+	if err != nil {
+		log.Infof("Agent did not find device plugins socket path dir %s: %v", socketDir, err)
+		return
+	}
+	if !stat.IsDir() {
+		log.Infof("Agent did not find valid device plugins socket path dir %s", socketDir)
```

Comment:
We should avoid logging anythin if the Agent is using the default value. You can use `config.IsConfigured` to know if a setting was set in any way by the user.